webpack/components/RolesIndex/RolesIndex.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+import { Table, TableHeader, TableBody } from '@patternfly/react-table';
+import { Pagination } from '@patternfly/react-core';
+import PropTypes from 'prop-types';
+import { linkCellFormatter, DEFAULT_PER_PAGE } from './RolesIndexConstants';
+
+const RolesIndex = ({ rowsData, RolesIndexactions }) => {
```

Comment:
nit: should we rename to `RolesIndexActions` ?

Reply:
Ops, yes